core/src/main/resources/jenkins/model/experimentalflags/UserExperimentalFlagsProperty/config.jelly

```diff
@@ -0,0 +1,74 @@
+<!--
+The MIT License
+
+Copyright (c) 2022, CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+    <j:set var="userProperty" value="${instance}"/>
+    <f:entry field="experimentalFlags">
+        <j:invokeStatic var="flagConfigs" className="jenkins.model.experimentalflags.UserExperimentalFlag"
+                        method="all"/>
+    
+        <j:if test="${empty(flagConfigs)}">
+            <div class="alert alert-info jenkins-form-item">
+                ${%NoFlagInfo}
```

Comment:
do we need to show this if there isn't any available? (seems unnecessary to me initially as there isn't anything for the user to do here, it's just space taken up for no reason, what do you think?)

Reply:
I thought it has an interest at the beginning or when no flags are active, to just let them know the section exists and it's expected that there is no items inside (could be confusing for users to have nothing inside).

Personally I don't really like the "script security approval" link, that appears only if you have something to approve. That could increase confusion when someone tells you what to look for, or if you want to find back what is the link.

Now, if you think it's better to not have it, I don't have a strong opinion, just personal preference 👍 

Do you think I should make the whole property disabled if there are no flag at the moment? (that's another option as well :D)

Reply:
We can try it out and see if we get any feedback, can always hide it later.